moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -157,6 +157,7 @@ ServoCalcs::ServoCalcs(rclcpp::Node::SharedPtr node,
 
   // Publish status
   status_pub_ = node_->create_publisher<std_msgs::msg::Int8>(parameters_->status_topic, ROS_QUEUE_SIZE);
+  condition_pub_ = node_->create_publisher<std_msgs::msg::Float64>("~/condition", ROS_QUEUE_SIZE);
```

Comment:
Would you mind moving the name of this topic into a constant in the anonymous namespace?  We generally try to avoid string constants and other magic numbers in the code.

Reply:
will do, thanks